torch/_inductor/ir.py

```diff
@@ -2990,6 +2990,8 @@ def generate_output(output, index=""):
                     packed,
                     index,
                 )
+            elif isinstance(output, int):
```

Comment:
@desertfire @ngimel Looks like this won't fly according to ci/cd

Reply:
I uncommented this out to make inductor tests happy, but This means that torch compile won't work with SDPA when fused kernels is used. 

The only other work (around that I can think) of is not returning ints from sdap_flash but rather return single dim 1 element tensors. But this feels bad. I am curious if I can avoid doing this